app/install/aws-linux.md

```diff
@@ -50,9 +45,9 @@ baseurl=https://kong.bintray.com/kong-rpm/amazonlinux/amazonlinux
     ```bash
     $ sudo yum update -y
     $ sudo yum install -y wget
-    $ wget https://bintray.com/kong/kong-rpm/rpm -O bintray-kong-kong-rpm.repo
-    $ sed -i -e 's/baseurl.*/&\/amazonlinux\/amazonlinux'/ bintray-kong-kong-rpm.repo
-    $ sudo mv bintray-kong-kong-rpm.repo /etc/yum.repos.d/
+    $ wget {{ site.links.download }}/kong-rpm/rpm -O kong.repo
+    $ sed -i -e 's/baseurl.*/&\/amazonlinux\/amazonlinux'/ kong.repo
+    $ sudo mv kong.repo /etc/yum.repos.d/
     $ sudo yum update -y
     $ sudo yum install -y kong
```

Comment:
What's the difference between this install section and the previous install section in the same topic?